frontend/app/src/components/profitloss/ReportActionableCard.vue

```diff
@@ -255,183 +249,149 @@ import { useAreaVisibilityStore } from '@/store/session/visibility';
-    const { report, isPinned } = toRefs(props);
-    const router = useRouter();
-    const { pinned } = storeToRefs(useAreaVisibilityStore());
-
-    const setDialog = (dialog: boolean) => {
-      emit('set-dialog', dialog);
-    };
-
-    const reportsStore = useReports();
-    const { actionableItems } = toRefs(reportsStore);
-
-    const actionableItemsLength = computed(() => {
-      let missingAcquisitionsLength: number = 0;
-      let missingPricesLength: number = 0;
-      let total: number = 0;
-
-      const items = get(actionableItems);
-
-      if (items) {
-        missingAcquisitionsLength = items.missingAcquisitions.length;
-        missingPricesLength = items.missingPrices.length;
-        total = missingAcquisitionsLength + missingPricesLength;
-      }
-
-      if (!missingAcquisitionsLength || !missingPricesLength) {
-        set(step, 1);
-      }
-
-      return {
-        missingAcquisitionsLength,
-        missingPricesLength,
-        total
-      };
-    });
+  isPinned: { required: false, type: Boolean, default: false }
+});
+
+const emit = defineEmits<{ (e: 'set-dialog', value: boolean): void }>();
+const { t, tc } = useI18n();
+const { report, isPinned } = toRefs(props);
+const router = useRouter();
+const { pinned } = storeToRefs(useAreaVisibilityStore());
+
+const setDialog = (dialog: boolean) => {
+  emit('set-dialog', dialog);
+};
+
+const reportsStore = useReports();
+const { actionableItems } = toRefs(reportsStore);
+
+const actionableItemsLength = computed(() => {
+  let missingAcquisitionsLength: number = 0;
+  let missingPricesLength: number = 0;
+  let total: number = 0;
+
+  const items = get(actionableItems);
+
+  if (items) {
+    missingAcquisitionsLength = items.missingAcquisitions.length;
+    missingPricesLength = items.missingPrices.length;
+    total = missingAcquisitionsLength + missingPricesLength;
+  }
+
+  if (!missingAcquisitionsLength || !missingPricesLength) {
+    set(step, 1);
+  }
 
-    const setPinned = (pin: Nullable<Pinned>) => {
-      set(pinned, pin);
-    };
-
-    const pinSection = () => {
-      const pinned: Pinned = {
-        name: 'report-actionable-card',
-        props: {
-          report: get(report),
-          isPinned: true
-        }
-      };
-
-      setPinned(pinned);
-      setDialog(false);
-    };
-
-    const step = ref<number>(1);
-
-    const stepperContents = computed(() => {
-      const contents = [];
-
-      const missingAcquisitionsLength = get(
-        actionableItemsLength
-      ).missingAcquisitionsLength;
-      if (missingAcquisitionsLength > 0) {
-        contents.push({
-          key: 'missingAcquisitions',
-          title: t('profit_loss_report.actionable.missing_acquisitions.title', {
-            total: missingAcquisitionsLength
-          }).toString(),
-          hint: t(
-            'profit_loss_report.actionable.missing_acquisitions.hint'
-          ).toString(),
-          selector: 'report-missing-acquisitions',
```

Comment:
this is not working. I think we can't use auto import component for this
just pass `selector: ReportMissingAcquisitions` here and import this component in this file.

Reply:
> Need some changes

Good catches, give me a bit to fix them.